app/interactors/send_team_hours_update.rb

```diff
@@ -0,0 +1,15 @@
+class SendTeamHoursUpdate
+  include Interactor
+  include HasHarvest
+
+  def call
+    Team.active.each do |team|
+      from = Date.this_week.first
+      to = Date.this_week.last
```

Comment:
I think what you have is fine. I'm just suggesting you move the `from` and `to` assignments out of the `each`, not out of the interactor or anything else.